andela/getmyshop
# 110078858 integrate delivery tracking for orders

#### What does this PR do?

It allows users to track their order.
#### Description of Task to be completed?

Add a delivery tracking support based on the time order was created. Add a cancel order button that the user can use to cancel order unless the order has been delivered.
#### How should this be manually tested?

When a user place an order, the user will be able to see the order status. The order can also be cancelled.
#### Any background context you want to provide?

There is currently no backend support for delivery tracking, so the current solution uses the time the order was placed to change the order status.
#### What are the relevant pivotal tracker stories?
#110078858 #113264733
